_projects/2024/100535153/100535153.Rmd

```diff
@@ -0,0 +1,941 @@
+---
+title: "The Changing Nature of Middle-Class Jobs"
+description: |
+  The types of jobs that pay middle-class wages — between $40,000 and $80,000 in 2014 dollars — have shifted since 1980. 
+  Fewer of these positions are in male-dominated production occupations, while a greater share are in workplaces more open to women.
+categories: "2024"
+author: Marco Boso
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="1000%", fig.align="center")
+```
+
+## Introduction
+
+The chart from The New York Times examines the shifting landscape of middle-class jobs, defined as positions paying between $40,000 and $80,000 in 2014 dollars, and how these changes have impacted the workforce since 1980. Over a span of 30 years, the analysis by a team of reporters and data journalists revealed a significant transformation in the types of jobs offering middle-class wages. Notably, there has been a decline in male-dominated production occupations, traditionally associated with manufacturing and physical labor. At the same time, a growing share of middle-class positions has emerged in workplaces that are more inclusive and accessible to women, such as healthcare, education, and administrative roles.
+
+This shift reflects broader societal changes, including advancements in technology, globalization, and evolving cultural norms about gender roles in the workforce. As part of a comprehensive series on the American middle class, the data highlights that women have been the primary beneficiaries of these shifts, gaining increased access to stable, well-paying jobs that were previously less available to them. 
+
+![The Changing Nature of Middle-Class Jobs - The New York Times](the-changing-nature-of-middle-class-jobs.png){.external}
+
+## Replication work
+
+### Load the packages 
+```{r packages}
+#| echo: true
```

Comment:
Please remove any "echo" option throughout the post. True is the default, and there should be no chunk with false, because we always want to show the code.